tests/query_test.go

```diff
@@ -87,6 +87,16 @@ func (s *QuerySuite) TestQuery() {
 	s.Panics(func() {
 		s.Equal("bar", store.MustFindOne(queryErr).Foo)
 	})
+
+	foobar := newQueryFixture("foo bar")
```

Comment:
Yes, they are fixed to `gopkg.in/src-d/go-kallax.v1`. What you can do is clone the repo at `$GOPATH/src/gopkg.in/src-d/go-kallax.v1` and add your own remote there. Unfortunately, until real package management is a thing in Go, we have to stick with such methodologies.